### ISSUE TITLE
Breaking-Down the Processor Technology

### ISSUE THREAD

**Issue by kvardhan10** [section-engineering-education/engineering-education#2209]
### Proposed title of article
Breaking-Down the Processor Technology

### Introduction paragraph (2-3 paragraphs):
Processor - one of the key components of a computer. Having the right CPU can benefit the user in doing the tasks faster and better. But it depends on various specifications and processor capabilities. This article will introduce some of the many features to the readers and help them understand how the computing capacity differs with various requirements.

### Key takeaways: 
After reading this article, the reader will have a clearer concept about the different capabilities of processors and will be able to differentiate between various types of processors available in the market right now.The reader will be introduced to various specifications/terms such as:
- Socket
- Cores and their types
- Clockspeed
- Cache Memory
- Threading and Hyperthreading

### References: 
https://www.pugetsystems.com/labs/articles/Specs-Explained-CPU-137/
https://techguided.com/7-cpu-specs-explained/

**Comment by hectorkambow** [comment]
to help avoid a backlog queue we are only accepting one article topic/pr approved at a time 👍

@kvardhan10  

We can re-open this one once the other one is merged 👍

**Comment by kvardhan10** [comment]
Hey @hectorkambow . 
Since my other article is merged now, can you re-open this issue?
or
Should I open a new issue with the same topic?
